cg/services/order_validation_service/models/case.py

```diff
@@ -7,11 +7,11 @@
 
 
 class Case(BaseModel):
-    data_delivery: DataDelivery
```

Comment:
On what level is data_delivery even sent? Since Cases don't exist in the FE request currently...